Bastion/src/isaac/bastion/manager/EnderPearlManager.java

```diff
@@ -42,12 +50,8 @@ public void handlePearlLaunched(EnderPearl pearl) {
 
 	private void getBlocking(EnderPearl pearl) {
 		double gravity = 0.03F;
-		try {
```

Comment:
yes, the try-catch was moved to the constructor so it only has to be done on startup to check if humbug is there